tst.h

```diff
@@ -70,4 +70,18 @@ char tst_get_key(const tst_node *node);
 unsigned tst_get_refcnt(const tst_node *node);
 char *tst_get_string(const tst_node *node);
 
+#define del_node(parent, victim, root, kid) \
```

Comment:
File `tst.h` was intended to be one of the public headers. However, macro `del_node` was implementation-specific, that prevented from being widely adopted. Move it into C source file rather than this header.

Reply:
Write down comment(s) for this macro.